src/LeanCode.ContractsGenerator/Compilation/ContractsCompiler.cs

```diff
@@ -30,11 +31,26 @@ public static class ContractsCompiler
         "System.Runtime.Reference",
         "System.Runtime.Extensions.Reference",
     });
+#else
+    public static readonly ImmutableHashSet<string> ReferenceAssemblyNames = ImmutableHashSet<string>.Empty;
+#endif
+
+#if NET7_0
```

Comment:
Confusing. Can we group them into a single `#if` so it's more obvious that each version has one set empty, different from the other version?